src/main.js

```diff
@@ -1,5 +1,26 @@
 import { createApp } from 'vue'
 import './style.css'
 import App from './App.vue'
+import {createRouter, createWebHistory} from 'vue-router';
+import Countries from '../views/Countries.vue'
+import DetailsPage from '../views/DetailsPage.vue'
 
-createApp(App).mount('#app')
+const routes = [
+    {
+        path: '/',
+        component: Countries,
+        name: 'Countries'
+    },
+    {
+        path: '/details',
+        component: DetailsPage,
+        name: 'DetailsPage'
```

Comment:
same as before but also change casing from `name: 'DetailsPage'` to `name: 'details-page'`